data/json/materials.json

```diff
@@ -746,7 +751,9 @@
     "dmg_adj": [ "marked", "dented", "smashed", "shattered" ],
     "bash_dmg_verb": "dented",
     "cut_dmg_verb": "scratched",
-    "burn_products": [ [ "scrap", 1 ] ]
+    "burn_products": [ [ "scrap", 1 ] ],
+    "compact_accepts": [ "budget_steel", "hardsteel", "iron" ],
+    "compacts_into": [ "steel_lump", "sheet_metal", "steel_chunk", "scrap" ]
```

Comment:
I am not sure `sheet_metal` is proper output.

Reply:
This preserves status quo: the original steel compactor also provided these four output options.

Reply:
Willing to remove `sheet_metal` (and by extension, `aluminum_foil`) if you insist -- let me know.